docs/source/advanced_topics/custom_menu_classes.rst

```diff
@@ -208,9 +208,9 @@ If you're happy with the default ``FlatMenu`` model, but wish customise the menu
         from django.db import models
         from django.utils.translation import ugettext_lazy as _
         from modelcluster.fields import ParentalKey
-        from wagtail.wagtailimages import get_image_model_string
-        from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
-        from wagtail.wagtailadmin.edit_handlers import FieldPanel, PageChooserPanel
+        from wagtail.iamges import get_image_model_string
```

Comment:
Just one small typo here

Reply:
Thanks! 😳

I've pushed an additional commit with the typo fixed.